Tests/Unit/Configuration/ConfigurationFinderTest.php

```diff
@@ -17,15 +17,35 @@
 use PHPUnit\Framework\Attributes\RunTestsInSeparateProcesses;
 use PHPUnit\Framework\Attributes\Test;
 use PHPUnit\Framework\TestCase;
+use TYPO3\CMS\Core\Configuration\ConfigurationManager;
+use TYPO3\CMS\Core\Configuration\Loader\YamlFileLoader;
 use TYPO3\CMS\Core\Core\ApplicationContext;
 use TYPO3\CMS\Core\Core\Environment;
+use TYPO3\CMS\Core\Log\Logger;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
 
 #[CoversClass(ConfigurationFinder::class)]
 #[RunTestsInSeparateProcesses]
 final class ConfigurationFinderTest extends TestCase
 {
     private static string $configPath;
 
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $configurationManager = new ConfigurationManager();
+        $GLOBALS['TYPO3_CONF_VARS'] = $configurationManager->getDefaultConfiguration();
+
+        GeneralUtility::addInstance(
+            YamlFileLoader::class,
+            GeneralUtility::makeInstance(
+                YamlFileLoader::class,
+                $this->createMock(Logger::class),
+            ),
+        );
```

Comment:
@brotkrueml This may could be done more elegant..
I got errors when running the phpunit test because of missing argument (Logger) for YamlFileLoader class (is instantiated in ConfigurationFinder.php L148. I was struggeling with the dependency injection of the LoggerAwareInterface for the YamlFileLoader class here and that was the solution i found which satisfied the test cases :)